02-standard-library/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "activity-monitor",
+    "version": "1.0.0",
+    "description": "A script to log system activity",
+    "main": "activityMonitor.ts",
+    "scripts": {
+      "start": "ts-node activityMonitor.ts"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "dependencies": {
```

Comment:
Usually these are saved as dev dependencies.Please consider changing them, to be in line with best practices... https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencie